src/main/java/DukeException.java

```diff
@@ -0,0 +1,36 @@
+//Implement a class DukeException that extends the Exception class. 
+//This class should be able to handle the different exceptions that may occur in your program.
+
+public class DukeException extends Exception {
+    //CommandException
+    // Constructor that accepts a message
+    public DukeException(String message)
+    {
+       super(message);
+    }
```

Comment:
The line breaks around curly brackets for constructors follow the same rule as method. It should be like this:
```
public DukeException(String message) {
    super();
}
```